Design Patterns/Design Patterns/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using AbstractFactory.Classes.Factories;
+using AbstractFactory.Interfaces;
+
+namespace AbstractFactory
+{
+    class Program
+    {
+        public static void Main(string[] args)
+        {
+            // Factory can be changed without any negative consequences to code
+            // This results in different behaviour with almost the same code
+            IAbstractCarFactory factory = new AbstractExtraFactory();
+
+            IAbstractEngine engine = factory.CreateEngine();
```

Comment:
The task is Implement a class that returns the desired set of parts, depending on the type of configuration. So i suppose that the factory should return model like ExtraCar, StandardCar etc with necessary types of wheels, engine and so one.

Reply:
Fixed. However, review is required as I doubt the solution I've implemented

Reply:
I think it is fine, just fix the naming please